Scripts/quicktest.out

```diff
@@ -0,0 +1,40 @@
+Should succeed
+
+Dafny program verifier finished with 1 verified, 0 errors
+Should fail
+/Users/davidcok/yucca/dafny/b.dfy(1,24): Error: assertion violation
```

Comment:
Add `-useBaseNameForFileName` to the call to Dafny to make this path just `b.dfy`.

Reply:
OK